backend/flow_api/flow/registration/action_register_login_identifier.go

```diff
@@ -110,23 +110,21 @@ func (a RegisterLoginIdentifier) Execute(c flowpilot.ExecutionContext) error {
 	}
 
 	// Decide which is the next state according to the config and user input
-	if email != "" && deps.Cfg.Emails.RequireVerification {
+	if email != "" && deps.Cfg.Identifier.Email.Verification {
 		if err := c.Stash().Set("passcode_template", "email_verification"); err != nil {
 			return fmt.Errorf("failed to set passcode_template to stash: %w", err)
 		}
 
 		if deps.Cfg.Password.Enabled {
 			return c.StartSubFlow(passcode.StatePasscodeConfirmation, StatePasswordCreation)
-		} else if !deps.Cfg.Passcode.Enabled || (deps.Cfg.Passkey.Onboarding.Enabled && c.Stash().Get("webauthn_available").Bool()) {
+		} else if deps.Cfg.Passkey.Onboarding.Enabled && c.Stash().Get("webauthn_available").Bool() {
 			return c.StartSubFlow(passcode.StatePasscodeConfirmation, passkey_onboarding.StateOnboardingCreatePasskey, shared.StateSuccess)
 		}
```

Comment:
When passwords and passkey onboarding are disabled, an `ErrorFlowDiscontinuity` would be returned instead of just going strait to passcode confirmation and after that to success. So I think an `else` block with `c.StartSubFlow(passcode.StatePasscodeConfirmation, shared.StateSuccess)` is missing.

Reply:
It will be an "email verification" as the else block should be inside the `if email != "" && deps.Cfg.Identifier.Email.Verification` block.